src/commons/eventsourcing/kv.rs

```diff
@@ -198,6 +205,13 @@ impl KeyValueStoreDiskImpl {
         path
     }
 
+    /// creates a file path, prefixing the name with '.' much like vi
+    fn swap_file_path(&self, key: &KeyStoreKey) -> PathBuf {
```

Comment:
What is meant by "swap file" in this context? I associate swap file with a specific data layout stored in a file or partition and used as "extra memory" by the operating system by way of swapping objects between main memory and swap storage. With a swap fs it is "safe" to lose the contents of the swap file, as safe as losing what is in main memory at least, but is this use of "swap" referring to a cache that is safe to lose or is it actually a permanent backing store that should not be lost?

Reply:
Ah I see "much like vi", so it is a temporary representation of edits that have not yet been finally persisted, and not actual final backing store?